config/notifiers.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/pkg/errors"
 	commoncfg "github.com/prometheus/common/config"
 	"github.com/prometheus/common/sigv4"
+
+	rcmodels "github.com/RocketChat/Rocket.Chat.Go.SDK/models"
```

Comment:
Since we don't use SDKs elsewhere in Alertmanager, it might be that you need to declare these models inline.